test/suite/test_backup14.py

```diff
@@ -79,13 +79,12 @@ def remove_data(self):
     def add_data_validate_backups(self):
         self.pr('Adding initial data')
         self.initial_backup = True
-        self.add_data(self.uri, self.bigkey, self.bigval)
+        self.add_data(self.uri, self.bigkey, self.bigval, do_checkpoint=True)
```

Comment:
`do_checkpoint=True` works as a solution to the checkpoint, but I would rather just have `True` and treat it as a bool instead of making an object.

Reply:
Updated in [1a6e19d](https://github.com/wiredtiger/wiredtiger/pull/7566/commits/1a6e19d385afcf807df8ef10114c1e0b59bc9982)